src/components/Settings/RouterPreferenceSettings/index.tsx

```diff
@@ -23,11 +22,6 @@ const InlineLink = styled(ThemedText.BodySmall)`
 export default function RouterPreferenceSettings() {
   const [routerPreference, setRouterPreference] = useRouterPreference()
   const dispatch = useAppDispatch()
-  const userOptedOutOfUniswapX = useUserOptedOutOfUniswapX()
-  const isUniswapXDefaultEnabled = useUniswapXDefaultEnabled()
-  const isUniswapXOverrideEnabled = isUniswapXDefaultEnabled && !userOptedOutOfUniswapX
-
-  const uniswapXInEffect = routerPreference === RouterPreference.X || isUniswapXOverrideEnabled
```

Comment:
this code implemented the overriding of the toggle display depending on the rollout phase. now that we're post-rollout, we can just rely directly on the `user.routerPreference` value